admin/modules/config/spiders.php

```diff
@@ -37,7 +37,7 @@
 		if(!$errors)
 		{
 			$new_spider = array(
-				"name" => $db->escape_string($mybb->input['name']),
+				"name" => $db->escape_string(htmlspecialchars($mybb->input['name'])),
```

Comment:
`htmlspecialchars_uni` should always be used. Additionally, sanitising should be done on output rather than input, so that the original data is stored in the DB.